Doc/whatsnew/3.15.rst

```diff
@@ -382,6 +382,13 @@ sqlite3
      (Contributed by Stan Ulbrych and Łukasz Langa in :gh:`133461`)
 
 
+socket
```

Comment:
Can you move this section before sqlite3?

Reply:
sure.